client/shared/schema.ts

```diff
@@ -6,34 +6,42 @@ export default {
   users: {
     type: 'object',
     required: ['id', 'permissions', 'first_name', 'last_name', 'email'],
+    description:
+      'Object that defines a user of Magic Wand. Either a partner or founder',
 
     properties: {
       id: { type: 'integer' },
       auth0: { type: 'string' },
-      email: { type: 'string' },
+      email: { type: 'string', format: 'email' },
       permissions: { type: 'string' },
-      first_name: { type: 'string' },
-      last_name: { type: 'string' },
-      school: { type: 'string' },
+      first_name: { type: 'string', maxLength: 128 },
+      last_name: { type: 'string', maxLength: 128 },
+      school: { type: 'string', maxLength: 128 },
```

Comment:
I generally avoid length limits unless there is some length limit inherent to the format.